internal/gojs/runtime.go

```diff
@@ -19,16 +20,21 @@ const (
 	getRandomDataName        = "runtime.getRandomData"
 )
 
+// Debug has unknown use, so stubbed.
+//
+// See https://github.com/golang/go/blob/go1.19/src/cmd/link/internal/wasm/asm.go#L133-L138
+var Debug = goarch.StubFunction("debug")
+
 // WasmExit implements runtime.wasmExit which supports runtime.exit.
 //
 // See https://github.com/golang/go/blob/go1.19/src/runtime/sys_wasm.go#L28
-var WasmExit = newSPFunc(wasmExitName, wasmExit)
-
-func wasmExit(ctx context.Context, mod api.Module, sp []uint64) {
-	mem := mod.Memory()
+var WasmExit = goarch.NewFunc(wasmExitName, wasmExit,
```

Comment:
this file defines the functions of GOARCH=wasm and now has no dependency on anything related to js